ucdavis_service/src/main/java/com/ucdavis/application/UCDavisService.java

```diff
@@ -140,6 +140,47 @@ public void writeToMap(List<JSONObject> records, String target) {
 		} else {
 			data.put(target, dataItems);
 		}
+		
+		checkNullKeysAndValue(dataItems, true);
+	}
+	
+	private void checkNullKeysAndValue(Map<String, Map<String,String>> dataItems, boolean isDaily) {
```

Comment:
suggest to have a general map null check, let me know your opinion on this. The general null check is basically taking any input map, it should work exactly like how we read the query and put it into the map

Reply:
So you suggest that the input should set as like Map<String, Obejct>,... ?